_sass/_partners.scss

```diff
@@ -0,0 +1,91 @@
+@media screen and (min-width: 280px) {
+    .each-partner {
+        width: 100%;
+        background-color: #273647;
+    }
+    .partner-logo{
+      width: 200px;
+      height: 8vw;
```

Comment:
The use of a defined width but a relative height on these two lines above is causing the proportions of the logos to rendered as skewed. Instead, you could simply remove these properties and set a width attribute only directly on the `img` elements in the markup.